index/scorch/segment/zap/merge.go

```diff
@@ -548,6 +547,8 @@ func writePostings(postings *roaring.Bitmap, tfEncoder, locEncoder *chunkedIntCo
 	return postingsOffset, nil
 }
 
+type VarintEncoder func(uint64) (int, error)
```

Comment:
this type probably doens't need to be exported, but we can do this in a follow-on change

Reply:
done!